lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -100,6 +100,15 @@ export class BrowserUtils {
         }
     }
 
+    /**
+     * Block redirectUri loaded in popup from calling AcquireToken APIs
+     */
+    static blockAcquireTokenInPopups(): void {
+        if (window.opener && window.opener !== window) {
```

Comment:
Two things:

1. I think you should pass in the interaction type, and only make this check if a popup request is being made, as redirect operations inside a popup should be allowed.
2. Make sure this test does result positives. Just beacuse a window has an opener doesnt mean its a popup.

Reply:
Yeah, this check caused problems in v1, and was part of the reason we put the interaction type in the state value. See https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/1413